include/fiction/technology/charge_distribution_surface.hpp

```diff
@@ -391,7 +391,25 @@ class charge_distribution_surface<Lyt, false> : public Lyt
         return strg->dist_mat[index1][index2];
     }
     /**
-     * Calculates and returns the electrostatic potential between two cells.
+     * Calculates and returns the chargeless electrostatic potential between two cells.
+     *
+     * @param c1 The first cell
+     * @param c2 The second cell
+     * @return The chargeless electrostatic potential between `c1` and `c2` (V_i,j / n_j).
+     */
+    [[nodiscard]] double get_chargeless_potential_between_sidbs(const typename Lyt::cell& c1,
+                                                                const typename Lyt::cell& c2) const noexcept
+    {
+        if (const auto index1 = cell_to_index(c1), index2 = cell_to_index(c2); (index1 != -1) && (index2 != -1))
+        {
+            return strg->pot_mat[static_cast<uint64_t>(index1)][static_cast<uint64_t>(index2)];
+        }
+
+        return 0;
+    }
+
+    /**
+     * Calculates and returns the electrostatic potential at one cell (c1) generated by another cell (c2).
```

Comment:
Same here: could you refer back to the other function in case you don't need the sign?